examples/compute_polar_coordinates.py

```diff
@@ -0,0 +1,403 @@
+ds_path = sample_data.fetch_dataset_paths(
+    "SLEAP_single-mouse_EPM.analysis.h5"
+)["poses"]
+ds = load_poses.from_sleap_file(ds_path, fps=None)  # force time_unit = frames
+
+print(ds)
+print("-----------------------------")
+print(f"Individuals: {ds.individuals.values}")
+print(f"Keypoints: {ds.keypoints.values}")
+
+
+# %%
+# The loaded dataset ``ds`` contains two data variables:``position`` and
+# ``confidence``. Both are stored as data arrays. In this tutorial, we will
+# use only ``position``:
+position = ds.position
+
+
+# %%
+# Compute head vector
+# ---------------------
+# To demonstrate how polar coordinates can be useful in behavioural analyses,
+# we will compute the head vector of the mouse.
+#
+# We define it as the vector from the midpoint between the ears to the snout.
+
+# compute the midpoint between the ears
+midpoint_ears = position.sel(keypoints=["left_ear", "right_ear"]).mean(
+    dim="keypoints"
+)
+
+# compute the head vector
+head_vector = position.sel(keypoints="snout") - midpoint_ears
+
+# drop the keypoints dimension
+# (otherwise the `head_vector` data array retains a `snout` keypoint from the
+# operation above)
+head_vector = head_vector.drop_vars("keypoints")
+
+# %%
+# Visualise the head trajectory
+# --------------------------------------
+# We can plot the data to check that our computation of the head vector is
+# correct.
+#
+# We can start by plotting the trajectory of the midpoint between the ears. We
+# will refer to this as the head trajectory.
+
+fig, ax = plt.subplots(1, 1)
+mouse_name = ds.individuals.values[0]
+
+sc = ax.scatter(
+    midpoint_ears.sel(individuals=mouse_name, space="x"),
+    midpoint_ears.sel(individuals=mouse_name, space="y"),
+    s=15,
+    c=midpoint_ears.time,
+    cmap="viridis",
+    marker="o",
+)
+
+ax.axis("equal")
+ax.set_xlabel("x (pixels)")
+ax.set_ylabel("y (pixels)")
+ax.invert_yaxis()
+ax.set_title(f"Head trajectory ({mouse_name})")
+fig.colorbar(sc, ax=ax, label=f"time ({ds.attrs['time_unit']})")
+fig.show()
+
+# %%
+# We can see that the majority of the head trajectory data is within a
+# cruciform shape. This is because the dataset is of a mouse moving on an
+# `Elevated Plus Maze <https://en.wikipedia.org/wiki/Elevated_plus_maze>`_.
+# We can actually verify this is the case by overlaying the head
+# trajectory on the sample frame of the dataset.
+
+# read sample frame
+frame_path = sample_data.fetch_dataset_paths(
+    "SLEAP_single-mouse_EPM.analysis.h5"
+)["frame"]
+im = plt.imread(frame_path)
+
+
+# plot sample frame
+fig, ax = plt.subplots(1, 1)
+ax.imshow(im)
+
+# plot head trajectory with semi-transparent markers
+sc = ax.scatter(
+    midpoint_ears.sel(individuals=mouse_name, space="x"),
+    midpoint_ears.sel(individuals=mouse_name, space="y"),
+    s=15,
+    c=midpoint_ears.time,
+    cmap="viridis",
+    marker="o",
+    alpha=0.05,  # transparency
+)
+
+ax.axis("equal")
+ax.set_xlabel("x (pixels)")
+ax.set_ylabel("y (pixels)")
+ax.invert_yaxis()
```

Comment:
I think there is no need for the y axis inversion this time. 
Because you plot the image first, matplotlib shows it in the "image" coordinate system (with y axis values increasing from top to bottom). When you overplot the trajectory on it, the "image" orientation is still maintained. Your axis inversion here undoes it and turns it the "wrong" way around. You can see that if you look at the y axis ticks of this vs the previous plot. If you verify this, maybe it's worth leaving a small comment to that effect (i.e. y axis inversion not needed if image is plotted first).